deploy.yaml

```diff
@@ -0,0 +1,27 @@
+#To use this you will need buildcomponents.yaml in your top level ci file. 
```

Comment:
Can you not pull buildcomponents.yaml into this file?

Reply:
Its very aggravating.  Per the docs (and our verification) includes only go one level deep. That means your gitlab ci file has to include everything, and the things it includes can't include anything.  It stinks.